### PR TITLE
fix(@clayui/css): Mixins check if parameter is map to avoid Sass error

### LINKED ISSUE #5120: @clayui/css: Some mixins aren't checking if parameter is type map
These will error if something other than a map is passed in.

```
clay-select-variant
clay-form-validation-variant
clay-custom-grid-columns
clay-generate-media-breakpoints
clay-after-highlight-variant
clay-input-group-stacked
clay-list-group-notification-item-variant
clay-loading-animation-variant
clay-menubar-vertical-expand
clay-modal-variant
clay-navbar-size
clay-slideout-variant
clay-sticker-size
clay-table-drag-variant
clay-table-clone-variant
clay-tbar-variant
clay-tbar-inline-down
clay-autofit-row
```

### PR DESCRIPTION
**Pull request by pat270**
fixes #5120

### CODE REVIEW

File: packages/clay-css/src/scss/mixins/_forms.scss [hunk 1]
```diff
@@ -701,282 +701,317 @@
...
-				),
-			background-size:
-				setter(map-get($map, bg-size), map-get($map, background-size)),
-			color:
-				setter(
-					map-get($map, color),
-					if(
-						variable-exists(input-color),
-						$input-color,
+	@if (type-of($map) == 'map') {
```
Comment (matuzalemsteles) [inline packages/clay-css/src/scss/mixins/_forms.scss:704]: Hey @pat270 can we do an early return to not change the indentation?

```sass
@if (type-of($map) != 'map') {
  @return
}
```

Comment (pat270) [reply packages/clay-css/src/scss/mixins/_forms.scss:704]: @matuzalemsteles good idea, unfortunately mixins don't return values. I can't think of another way around it atm.

Comment (matuzalemsteles) [reply packages/clay-css/src/scss/mixins/_forms.scss:704]: Oh yes that's true, that's ok, let's go with it!

**Review by matuzalemsteles — APPROVED** [review]
LGTM

**Pull request merged**